submissions/drewzag/hooli-style-popup/style.css

```diff
@@ -0,0 +1,158 @@
+*,
+*::after,
+*::before {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+a,
+a:link,
+a:visited,
+a:hover {
+  text-decoration: none;
+  color: gray;
+}
+
+.popup-image {
+  max-height: 2em;
+}
+
+.popup {
+  cursor: pointer;
+  height: 1.7em;
+}
+
+.bell {
+  height: 1.5em;
+}
+
+.header {
+  background-color: bisque;
+  padding: 10px 100px 0px 25px;
+  height: 3em;
+}
+
+.nav {
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)